spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -379,17 +378,35 @@ def compute_amplitudes_cutoff(waveform_extractor, peak_sign='neg',
 
     extremum_channels_ids = get_template_extremum_channel(waveform_extractor, peak_sign=peak_sign)
 
+    spike_amplitudes = None
+    invert_amplitudes = False
+    if waveform_extractor.is_extension("spike_amplitudes"):
+        amp_calculator = waveform_extractor.load_extension("spike_amplitudes")
+        spike_amplitudes = amp_calculator.get_data(outputs="by_unit")
+        if amp_calculator._params["peak_sign"] == "pos":
+            invert_amplitudes = True
+    else:
+        warnings.warn(
+            "'spike_amplitude' extension not found. Amplitudes will be computed using waveforms (by default 500), "
+            "which might lead to mis-estimation of amplitude cutoff. Pre-computing amplitudes for all spikes is "
+            "strongly recommended: `spikeinterface.postprocessing.compute_spike_amplitudes(waveform_extractor)`"
+        )
```

Comment:
I am not sure of this warning.
this will be super annoyig.
it was the previous behavior.
Maybe a warning if not enough sipkes but not every times.

Reply:
no this is needed. or we raise an error or skip it